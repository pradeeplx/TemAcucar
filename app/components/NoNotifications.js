import React from 'react'
import { fontFactor } from "../helpers"
import Colors from "../Colors"
import Sentence from "./Sentence"

export default NoNotifications = () => (
  <Sentence style={{
    alignSelf: 'stretch',
    margin: 10,
    textAlign: 'center',
    fontSize: 10 * fontFactor(),
  }}>
    Nenhuma novidade ainda. Que tal comeÃ§ar a interagir agora mesmo fazendo um pedido?
  </Sentence>
)
