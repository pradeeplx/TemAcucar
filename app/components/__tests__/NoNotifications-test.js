import 'react-native'
import React from 'react'
import NoNotifications from '../NoNotifications'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <NoNotifications />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
