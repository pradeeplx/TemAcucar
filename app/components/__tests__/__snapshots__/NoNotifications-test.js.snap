exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Object {
        "alignSelf": "stretch",
        "fontSize": 15,
        "margin": 10,
        "textAlign": "center",
      },
    ]
  }>
  Nenhuma novidade ainda. Que tal comeÃ§ar a interagir agora mesmo fazendo um pedido?
</Text>
`;
